_tools/types/main.go

```diff
@@ -0,0 +1,341 @@
+// Generates a report about every driver wich UASTv2 types does it use,
+// both in actual test fixtures and though the source code.
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"reflect"
+	"regexp"
+	"strings"
+	"sync"
+
+	"github.com/bblfsh/sdk/driver/manifest/discovery"
+	"github.com/bblfsh/sdk/uast"
+
+	_ "net/http/pprof"
+)
+
+const (
+	fixtureDir = "fixtures"
+	fixtureExt = ".sem.uast"
+	goDocURL   = "https://godoc.org/github.com/bblfsh/sdk/uast#"
+)
+
+var (
+	reposRootPath = path.Join(".", "drivers")
```

Comment:
For most uses, we should prefer `filepath.Join` (and the [filepath](https://godoc.org/path/filepath) package in general) for paths that are to be used with a real filesystem.

The difference is that `path` handles slash-separated paths, whereas `filepath` handles system-specific path layout. Since we do most everything on Linux this won't really make a difference in the observed results, but it follows the intended semantics of the package.